infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/Constants.java

```diff
@@ -37,5 +37,11 @@
   public static final String MACHINE_NAME_ANNOTATION_FMT =
       "org.eclipse.che.container.%s.machine_name";
 
+  /**
+   * The additional property name with the wildcard reserved for workspace id. Formatted property
+   * with the real workspace id is used to get workspace subpaths directories.
```

Comment:
Please document which format value has, and that this value will be removed before PVC creation